update.sh

```diff
@@ -16,4 +16,4 @@ errexit () {
 export OPENSHIFT_ROLE_FILTER=master,image
 
 cd provision-cluster
-./ansible-playbook.sh update-node-image.yml $1
+./ansible-playbook.sh update.yml $1
```

Comment:
There will need to be a update.yml playbook to make this change... It can just start as just using:
```
---
- import_playbook: update-node-image.yml
```
Later, once we have figured out how to do a rolling update of the masters we can add that to the update playbook as well.